packages/extension-ui/src/components/Address.tsx

```diff
@@ -163,10 +165,9 @@ function Address({
 
   const _onCopy = useCallback(() => show(t('Copied')), [show, t]);
 
-  // TODO: this will be reused
-  // const _toggleVisibility = useCallback((): void => {
-  //   address && showAccount(address, isHidden || false).catch(console.error);
-  // }, [address, isHidden]);
+  const _toggleVisibility = useCallback((): void => {
+    address && showAccount(address, isHidden || false).catch(console.error);
```

Comment:
I know it might seem clever and concise but if this is not a return value please use normal `if` statement for readability :)